packages/flutter_tools/lib/src/android/android_studio_validator.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
nits: there's more style nits in subsequent files but hopefully you can extrapolate from the earlier comments. Please use Flutter style throughout. Thanks.